src/rez/cli/rm.py

```diff
@@ -15,6 +15,11 @@ def setup_parser(parser, completions=False):
         "-p", "--package",
         help="remove the specified package (eg 'foo-1.2.3'). This will work "
         "even if the package is currently ignored.")
+    group.add_argument(
+        "-f", "--family",
+        help="remove the entire specified package family (eg 'python'). This "
+        "is only supported if the family is empty, or all its packages are "
+        "hidden")
```

Comment:
I think we ought to avoid archiving a family if it contains any packages, including hidden packages.

Reply:
agreed, updating